rosidl_generator_java/resource/msg.cpp.em

```diff
@@ -76,10 +77,15 @@ for member in message.structure.members:
             include_prefix = include_prefix[:-8]
         elif include_prefix.endswith('__feedback'):
             include_prefix = include_prefix[:-10]
+        elif include_prefix.endswith('__send_goal'):
+            include_prefix = include_prefix[:-11]
+        elif include_prefix.endswith('__get_result'):
+            include_prefix = include_prefix[:-12]
```

Comment:
__send_goal and __get_result are suffixes for services names so they shouldn't be needed here, right?

Reply:
We could probably remove them here, since I don't think it's currently possible for a service (or the request/response message of a service) to be a member of a message definition. But I'd rather leave it here just in case.